.github/workflows/PR-pipeline.yml

```
@@ -0,0 +1,145 @@
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v5
+        with:
+          node-version: 20.11.0
+
+      # ESLint and Prettier must be in `package.json`
+      - name: Install Node.js dependencies
+        run: npm ci
+
+      - name: Run linters
+        uses: wearerequired/lint-action@v2
```

actionlint (1.7.8)

the runner of "wearerequired/lint-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)